docs/faq/items/black_api.md

```diff
@@ -0,0 +1,4 @@
+# Does Black have an API?
+
+No. _Black_ is a command line tool. Many [integrations](../../integrations/index.rst)
+are provided, but a Python interface is not one of them.
```

Comment:
We can add that we *are* planning to add some API in the future.

Reply:
Okay! Any details I should add with that?

Reply:
We talked about how the API would probably be basically `black.format_str()`, but I'm not sure we should commit to that in public. I think we should just say that we will at some point add an API and link to https://github.com/psf/black/issues/779.